src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,59 @@
+import ClassNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as currentTodo } from '../../features/currentTodo';
```

Comment:
change name (IMO `actions` is not bad): you have `state.currentTodo` and `currentTodo` as object of actions, which could be a bit confusing